lib/twind/config.ts

```diff
@@ -42,23 +42,31 @@ export const config = defineConfig({
       colors: {
         // TODO: custom helpwave (hw) colors go here
```

Comment:
I used these same colors [here](https://github.com/JannikWibker/docs/blob/main/lib/twind/config.ts) for testing around a bit, copy them from there ideally.

The advantage of that is that all the colors are neatly separated into 3 groups: base colors, light colors and dark colors